.circleci/config.yml

```
@@ -11,7 +11,7 @@ jobs:
           command: /opt/python/cp27-cp27m/bin/pip install numpy scipy "pytest<3.7" pytest-astropy pytest-xdist Cython jinja2
       - run:
           name: Run tests
-          command: PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --parallel=4 || (git reset HEAD && PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --parallel=4)
+          command: PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --offline --no-git --parallel=4 || (git reset HEAD && PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --parallel=4)
```

maybe also try to remove my none working magic, too?

```suggestion
          command: PYTHONHASHSEED=42 /opt/python/cp27-cp27m/bin/python setup.py test --offline --no-git --parallel=4 
```